### PR TITLE
Brownies!

### PR DESCRIPTION
**Pull request by Bierkraan**
[feature] [add to wiki]

## About the PR 
adds some brownies to the game with terible sprites

## Why's this needed? 
on floorpills request

## Changelog

```changelog
(u)Bierkraan
(*)Added brownies in all kinds of forms
```

### CODE REVIEW

**Comment by Bierkraan** [comment]
wait let me fix that

File: code/datums/manufacturing.dm [hunk 1]
```diff
@@ -1935,6 +1935,7 @@ proc/get_nice_mat_name_for_manufacturers(mat)
 /datum/manufacture/hat_ltophat
 	name = "Large Top Hat"
 	item_paths = list("FAB-1")
+	item_names = list("Fabric")
```
Comment (ZeWaka) [inline code/datums/manufacturing.dm:1938]: Your branch is fucked up

Comment (Bierkraan) [reply code/datums/manufacturing.dm:1938]: Shit, Will het to fixing it tomorrow

Comment (BenLubar) [reply code/datums/manufacturing.dm:1938]: Try `git rebase -i master` and then delete all the lines with the commits that shouldn't be in the branch.

File: code/modules/food_and_drink/cakes.dm [hunk 2]
```diff
@@ -216,3 +216,58 @@
...
+	icon_state = "cannabis_brownie"
+	amount = 3
+	heal_amt = 2
+	w_class = 3
+	initial_reagents = list("chocolate"=3, "THC"=3)
+	food_effects = list("food_energized")
+
+/obj/item/reagent_containers/food/snacks/life_brownie
+	name = "lifeweed brownie"
+	desc = "Let's be honest, this one of the more usefull things the chef could make."
```
Comment (ZeWaka) [inline code/modules/food_and_drink/cakes.dm:244]:
```suggestion
	desc = "Let's be honest, this one of the more useful things the chef could make."
```

File: code/datums/manufacturing.dm (hunk 1 above)
Comment (ZeWaka) [inline code/datums/manufacturing.dm:1938]:
```suggestion
```

File: code/datums/cookingrecipes.dm [hunk 3]
```diff
@@ -1340,6 +1340,40 @@
...
+	item3 = /obj/item/plant/herb/cannabis/black
+	cookbonus = 14
+	output = /obj/item/reagent_containers/food/snacks/death_brownie
+weed
+/datum/cookingrecipe/rainbow_brownie
+	item1 = /obj/item/reagent_containers/food/snacks/ingredient/dough_s
+	item2 = /obj/item/reagent_containers/food/snacks/condiment/chocchips
+	item3 = /obj/item/plant/herb/cannabis/mega
+	cookbonus = 14
+	output = /obj/item/reagent_containers/food/snacks/rainbow_brownie
```
Comment (ZeWaka) [inline code/datums/cookingrecipes.dm:1369]: can you move these above, to reflect the order used in the recipe hander

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Pull request closed without merging**